### PR TITLE
Support for Entities with constructor parameters

### PR DESCRIPTION
**Pull request by alulord**
This will allow to crate fixtures for classes, that expects constructor parameters e.g. FOS\UserBundle\Model\Group

### CODE REVIEW

File: Tool/FixtureGenerator.php [hunk 1]
```diff
@@ -325,9 +325,29 @@ public function generateFixtureItemStub($item)
...
+                if ($reflexion->hasMethod('get'.ucfirst($paramName))) {
+                    $constructorParams[] = $item->{'get'.ucfirst($paramName)}();
+                } elseif ($reflexion->hasProperty($paramName)) {
+                    $reflectionProperty = $reflexion->getProperty($paramName);
+                    $reflectionProperty->setAccessible(true);
+                    $constructorParams[] = $reflectionProperty->getValue($item);
+                }
+            }
+        }
+        if (!empty($constructorParams)) {
```
Comment (Webonaute) [inline Tool/FixtureGenerator.php]: The change increase the method complexity.
please move code filling $constructorParams into its own method.
$constructorParams = $this->getConstructorParams(...);

**Comment by alulord** [comment]
@Webonaute I did refactor it a bit, hope it's ok.

**Comment by alulord** [comment]
I just found out, that my change will run correctly, however it will not create a correct fixture. There has to be a change in \Webonaute\DoctrineFixturesGeneratorBundle\Tool\FixtureGenerator::generateFixture

**Comment by alulord** [comment]
Should be ok now

**Pull request merged**